projects/packages/publicize/src/class-publicize-base.php

```diff
@@ -1433,6 +1433,18 @@ protected static function build_sprintf( $args ) {
 		}
 		return str_replace( $search, $replace, $string );
 	}
+
+	/**
+	 * Get Calypso URL for Publicize connections.
+	 *
+	 * @param string $source The idenfitier of the place the function is called from.
+	 * @return string
+	 */
+	public function publicize_calypso_url( $source = null ) {
```

Comment:
Should we change the name of this function as well? Maybe something like `connections_url` or `connection_settings_url`?